core/src/main/java/com/digitalpebble/storm/crawler/filtering/regex/RegexURLNormalizer.java

```diff
@@ -42,14 +48,10 @@
 import org.w3c.dom.Text;
 import org.xml.sax.InputSource;
 
-import com.digitalpebble.storm.crawler.Metadata;
-import com.digitalpebble.storm.crawler.filtering.URLFilter;
-import com.fasterxml.jackson.databind.JsonNode;
```

Comment:
Hum... Not sure storm-crawler has the same import ordering rule. You need to override your Eclipse settings so storm-crawler-core project keeps using the original/default formatting rules